regreg/affine/tests/test_factored_matrix.py

```diff
@@ -5,10 +5,9 @@
 from regreg.atoms.projl1_cython import projl1
 from regreg.api import identity_quadratic
 from regreg.atoms.tests.test_seminorms import all_close
+from regreg.tests.decorators import set_seed_for_test
 
-X = np.random.standard_normal((100, 50))
```

Comment:
You probably know, but another standard way of doing this is:

```
rng = np.random.RandomState(seed_value)
X = rng.standard_normal((100, 50))
```

Then you use `rng.<method>` instead of `np.random.<method>` in the rest of the test module.

This works because nose does I think guarantee the order of the tests being run, so the input is predictable for an individual test.   This also preserves the global random state.